examples/ffmpeg/ffmpeg/operations.py

```diff
@@ -1,35 +1,28 @@
 import asyncio
 
 from dffml.df.base import op
-from .definitions import input_file,output_file
+from .definitions import input_file, output_file
 
 
-@op(
-    inputs = {
-        "input_file" : input_file,
-        "output_file" : output_file
-    },
-    outputs = {
-    }
-)
-async def convert_to_gif(input_file,output_file):
+@op(inputs={"input_file": input_file, "output_file": output_file}, outputs={})
+async def convert_to_gif(input_file, output_file):
     # TODO
     # add start and end times in second version and rebuild using webhook
```

Comment:
We probably want to make the resolution (1920) an input as well. For when we deploy the dataflow we can add these to the seed values